PayPal_Braintree/styles/modules/_google-pay.scss

```diff
@@ -0,0 +1,183 @@
+button.braintree-googlepay-button {
+    background-origin: content-box;
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    border: 0;
+    border-radius: $spacer--extra-small;
+    box-shadow: rgba(60, 64, 67, 0.3) 0 1px 1px 0, rgba(60, 64, 67, 0.15) 0 1px 3px 1px;
+    cursor: pointer;
+    height: 40px;
+    min-height: 40px;
+    padding: $spacer--small $spacer--semi-medium;
+}
```

Comment:
I'd create variables for this file, might be in the same file but to be easy to change some value, as:
`box-shadow`, `border-radius`,  `height` & `paddings`